java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/entree/EntreeOrdered.java

```diff
@@ -0,0 +1,14 @@
+package com.my.FoodTruckApp.entree;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class EntreeOrdered {
+    private Integer orderId;
+    private Integer entreeId;
+}
+//when creating models named after a table, name it the same thing
```

Comment:
is there a better place for this note/doc?

Reply:
It has been moved to a notes doc